src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -530,6 +519,26 @@ export default class FilingHistoryList extends Vue {
       const effectiveDate = new Date(filing.effectiveDate)
       const submittedDate = new Date(filing.submittedDate)
 
+      // *** TODO: remove after debugging
+      // if (EnumUtilities.isTypeAdminFreeze(filing) && (filing.data?.adminFreeze?.freeze === true)) {
+      //   // change this to a Limited Restoration Extension filing for testing
+      //   filing = {
+      //     ...filing,
+      //     name: FilingTypes.RESTORATION,
+      //     displayName: 'Limited Restoration Extension Application',
+      //     filingSubType: FilingSubTypes.RESTORATION_EXTENSION
+      //   }
+      // }
+      // if (EnumUtilities.isTypeAdminFreeze(filing) && (filing.data?.adminFreeze?.freeze === false)) {
+      //   // change this to a Limited Restoration Conversion filing for testing
+      //   filing = {
+      //     ...filing,
+      //     name: FilingTypes.RESTORATION,
+      //     displayName: 'Conversion To Full Restoration Application',
+      //     filingSubType: FilingSubTypes.RESTORATION_CONVERSION
+      //   }
+      // }
```

Comment:
This is to "fake" restoration extension and conversion filings for development purposes, since the Legal API doesn't return this data yet. I will delete this eventually.